example/Doc.tsx

```diff
@@ -73,13 +76,13 @@ const Doc = () => {
       };
 
       try {
-        // @ts-ignore
         const code = await mdx.compile(doc, { ...opts, components: componentsByExport, useTailwind: true });
         const content = await mdx.run(code, { components: executedComponents, terms, variables });
 
         setError(() => null);
         setContent(() => content);
       } catch (e) {
+        // eslint-disable-next-line no-console
         console.error(e);
```

Comment:
would **love** if the markdown library had a pluggable logging system

Reply:
What are you imagining?